aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1644,6 +1614,91 @@ static at::Tensor _quantized_convolution_onednn(
     unary_scalars,
     unary_algorithm.has_value() ? unary_algorithm.value() : ""
   );
+
+#if IDEEP_PREREQ(3, 1, 0, 0)
+  // Use oneDNN's APIs instead of prepare/compute from ideep to reduce framework overhead.
+  // oneDNN version >= 3.1.0 is required.
+  using ideep::tensor;
+  static tensor empty_tensor;
+  bool has_postop_add = has_binary_post_op && binary_attr.value() == "add";
```

Comment:
I think qconv with binary add is not supported yet. Did you mean also add this new post op support in this PR beside reduce the framework overhead?

Reply:
No, I didn't add `add`. I have removed it. Thanks.